client/TextArea.jsx

```diff
@@ -74,9 +74,10 @@ const TextArea = () => {
         />
         <div onChange={set("submission")}>
           <label htmlFor="submission">
```

Comment:
Bug: You don't need this label any longer, since it's not attached to any particular input. In addition, to my knowledge, you don't want to have multiple inputs which have their own labels inside another label. I'd remove ln76 and ln81. Instead, here you could put a heading with some meaningful text (I assume it would be an `h2`, since I only see an `h1` coming before this). This would allow people on assistive technology to navigate by heading and quickly find the offers area.

Reply:
Do you mean change this label to h2?

Reply:
I'd suggest to change it to whatever heading level is appropriate at this point in the document.